### PR TITLE
Feature/keystone avg scores

### PR DESCRIPTION
**Pull request by WFrancois**
- Add actual data
- Remove the use of the current affix
- Match more keystone level

### CODE REVIEW

File: RaiderIO.toc [hunk 1]
```diff
@@ -9,6 +9,7 @@
 db/db_regions.lua
 db/db_realms.lua
 db/db_score_tiers.lua
+db/db_scorestats.lua
```
Comment (jahraphael) [inline RaiderIO.toc]: For consistency let's rename this to `db_score_stats.lua`

File: core.lua [hunk 2]
```diff
@@ -310,11 +310,22 @@ do
...
+		local level = 0;
+		for i, regex in ipairs(regexesFindLevel) do
+			level = raw:match(regex);
+			level = tonumber(level)
+			if level and level < 32 then
+				break
+			end
+		end
+
+		if not level or level < 1 then
```
Comment (jahraphael) [inline core.lua]: Since M+ only begins at 2 we could skip if level < 2 here

Comment (WFrancois) [reply core.lua]: Yeah, I mainly put it to avoid 0 value with the tonumber, but can be use for that as well.

**Comment by jahraphael** [comment]
Looks good :) Thanks for the work! This is merged in now.

**Pull request merged**